lib/trailblazer/activity/path.rb

```diff
@@ -81,12 +81,27 @@ def sequence_insert_options
           }
         end
 
+        def normalize_duplications((ctx, flow_options), *)
+          return Right, [ctx, flow_options] if ctx[:replace]
+
+          signal, (ctx, flow_options) = raise_on_duplicate_id([ctx, flow_options])
+          signal, (ctx, flow_options) = clone_duplicate_activity([ctx, flow_options])
+
+          return signal, [ctx, flow_options]
+        end
+
         def raise_on_duplicate_id((ctx, flow_options), *)
-          id, sequence, insert = ctx[:id], ctx[:sequence], ctx[:sequence_insert][0] # DISCUSS: should we use Replace here or rather the option(s)?
```

Comment:
@apotonick About `DISCUSS: should we use Replace here or rather the option(s)?` note, I preferred using `options` because of a small difference - `Linear ::Insert.method(:Replace)` creates new object every time. 
Let me know otherwise.